File: Java/1228.java
import java.util.Scanner;
import java.io.*;

public class Main{
	public static void main(String[] args) throws Exception{
		Scanner sc = new Scanner(System.in);
		BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out, "utf-8"));
		
		double a = sc.nextDouble();
		double b = sc.nextDouble();
		
		double std = (a - 100) * 0.9;
		double bimando = (b - std) * 100 / std;
		
		if(bimando <= 10) bw.write("정상");
		else if(bimando <= 20) bw.write("과체중");
		else bw.write("비만");
		
		bw.flush();
	}
}

File: C/1228.c
#include <stdio.h>

int main(){
	double a, b;
	scanf("%lf %lf", &a, &b);
	
	double std = (a - 100) * 0.9;
	double bimando = (b - std) * 100 / std;
	
	if(bimando <= 10) printf("정상");
	else if(bimando <= 20) printf("과체중");
	else printf("비만");
	
	return 0;
}